analytics/analytics.go

```diff
@@ -0,0 +1,92 @@
+package analytics
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/bitrise-io/bitrise/bitrise"
+	"github.com/bitrise-io/bitrise/configs"
+	"github.com/bitrise-io/bitrise/models"
+)
+
+//=======================================
+// Consts
+//=======================================
+
+const analyticsURL = "https://bitrise-stats.herokuapp.com/save"
+
+//=======================================
+// Models
+//=======================================
+
+// AnonymizedUsageModel ...
+type AnonymizedUsageModel struct {
+	ID      string        `json:"step"`
+	Version string        `json:"version"`
+	RunTime time.Duration `json:"duration"`
+	Error   bool          `json:"error,omitempty"`
+}
+
+// AnonymizedUsageGroupModel ...
+type AnonymizedUsageGroupModel struct {
+	Steps []AnonymizedUsageModel `json:"steps"`
+}
+
+//=======================================
+// Main
+//=======================================
+
+// SendAnonymizedAnalytics ...
+func SendAnonymizedAnalytics(buildRunResults models.BuildRunResultsModel) error {
+	if configs.OptOutUsageData == true {
```

Comment:
`configs.OptOutUsageData` should be `configs.IsAnalyticsOptOut`, and no need to `== true`